lesson_06/homeworks_06.py

```diff
@@ -56,17 +128,107 @@
 guesses = 0
 max_guesses = 5
 print("Вгадайте число від 1 до 20 за 5 спроб!")
+for i in range(max_guesses):
+    while True:
+        try:
+            selected_number = int(input('Введіть ціле число від 1 до 20 '))
+            if 1 <= selected_number <= 20:
+                break
+            else:
+                print('Число має бути від 1 до 20')
+        except ValueError:
+            print('Будь ласка, вводьте ціле число.')
+    guesses += 1
+    if selected_number == secret_number:
+        print(f'Вітаю! Ви вгадали число за {guesses} сспроби')
+        break
+    else:
+        print(f'Ви можете спробувати ще {max_guesses - guesses} разів')
+else:
+    print("Ви використали усі 5 спроб. Пощастить іншим разом!")
+print()
 
-# task 9
+print('task 9')
 """  Задача з використанням циклу for та continue. Задано список фруктів 'fruits'
 потрібно вивести на екран всі елементи списку, окрім "orange".
 """
 fruits = ["apple", "banana", "orange", "grape", "mango"]
+excepted_fruit = 'orange'
+for i in fruits:
+    if  i == excepted_fruit:
+        continue
+    print(i)
+print()
 
-# task 10
+print('task 10')
 """  Задано список чисел numbers, потрібно знайти список квадратів
 парних чисел зі списку. Спробуйте використати if та цикл for в один рядок.
 """
 numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-result = ["Відповідь вставте сюди"]
+result = [i**2 for i in numbers if i % 2 == 0]
 print(result)  #  [4, 16, 36, 64, 100]
+print()
+
+print('task 6.1 from LMS')
+"""Порахувати кількість унікальних символів в строці. Якщо їх більше 10 - 
+вивести в консоль True, інакше - False. Строку отримати за допомогою функції input()
+Перевірте, що репозиторій в git є публічним або ваш вчитель доданий до цього репозиторію.
+Ви можете перевірити це в інкогніто-режимі вашого браузера.
+Просто вставте посилання на свій репозиторій та переконайтеся, що ви можете побачити код."""
+character_count = 0
+input_phrase = input('Напишіть будь-яке слово чи речення.')
+for i in input_phrase:
+    character_count += 1
+print(character_count > 10)
+print()
+
+print('task 6.2 from LMS')
+"""Напишіть цикл, який буде вимагати від користувача ввести слово, в якому є літера "h" 
+(враховуються як великі так і маленькі). Цикл не повинен завершитися, якщо користувач ввів слово без букви "h".
+Перевірте, що репозиторій в git є публічним або ваш вчитель доданий до цього репозиторію.
+Ви можете перевірити це в інкогніто-режимі вашого браузера.
+Просто вставте посилання на свій репозиторій та переконайтеся, що ви можете побачити код."""
+while True:
+    input_phrase = input('Write any word that contains the letter "h".')
+    count = 0
+    for i in input_phrase:
+        if i == 'h' or i == 'H':
+            count += 1
+    if count >= 1:
+        print(f"Thanks, for  your word '{input_phrase}'")
+        break
```

Comment:
```suggestion
    if 'h' in input_phrase or 'H'in input_phrase:
    print(f"Thanks, for  your word '{input_phrase}'")
    break
```
@alex-pancho 
Мені здається так більш лаконічно та вірно